lib/paper_trail/reifier.rb

```diff
@@ -162,6 +164,15 @@ def reify_has_ones(transaction_id, model, options = {})
         end
       end
 
+      # Suppress #save so we can reassociate with the reified master of a has_one relationship
```

Comment:
Can you expand on this comment? Specifically, please explain what changed in AR5.

Also, please alphabetize methods.